api/workAllocation2/index.ts

```diff
@@ -46,6 +46,7 @@ import {
 taskServiceMock.init();
 caseServiceMock.init();
 roleServiceMock.init();
+roleServiceMock.mockDelete();
```

Comment:
when the mock method is put back to init(), this line can be removed.

Reply:
Done